site/content/en/docs/concepts/workload_priority_class.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Workload Priority Class"
+date: 2023-10-02
+weight: 6
+description: >
+  A priorityClass which value is utilized by Kueue controller and is independent from pod's priority.
+---
+
+Sometimes developers would like to control workload's priority without affecting pod's priority, like these cases:
```

Comment:
Use the second person:
A WorkloadPriorityClass allows you to control the Workload's priority without affecting the pod's priority.

Reply:
Also link to the documentation for Workload.